OpenRA.Mods.Common/Widgets/Logic/ModBrowserLogic.cs

```diff
@@ -96,6 +84,23 @@ public ModBrowserLogic(Widget widget)
 			RebuildModList();
 		}
 
+		void LoadModImages(ModMetadata mod, Dictionary<string, Sprite> destinationByModId, string path, Size requiredSize)
+		{
+			try
+			{
+				using (var image = new Bitmap(path))
+					if (image.Size == requiredSize)
+					{
```

Comment:
The `if` has `{ }`, so OpenRA regulations state that the outer `using` must have as well.